test/SILOptimizer/address_lowering.sil

```diff
@@ -938,20 +938,20 @@ bb0(%0 : @owned $P):
 }
 
 // CHECK-LABEL: sil [ossa] @f161_testOpenedArchetype : $@convention(thin) (@in any P) -> () {
-// CHECK: bb0(%0 : $*any P):
```

Comment:
ok, but %0 is the most concise and recognizable way to refer to the first function argument.

Reply:
Switched this function back to the way it was for this register.